third_party/move/move-core/types/src/value.rs

```diff
@@ -9,19 +9,29 @@ use crate::{
     u256,
 };
 use anyhow::{anyhow, bail, Result as AResult};
+use once_cell::sync::Lazy;
 use serde::{
-    de::{Error as DeError, Unexpected},
-    ser::{SerializeMap, SerializeSeq, SerializeStruct, SerializeTuple},
+    de::{EnumAccess, Error as DeError, VariantAccess},
+    ser::{SerializeMap, SerializeSeq, SerializeStruct, SerializeTuple, SerializeTupleVariant},
     Deserialize, Serialize,
 };
 use std::{
+    collections::{btree_map::Entry, BTreeMap},
     convert::TryInto,
     fmt::{self, Debug},
+    sync::Mutex,
```

Comment:
nit: use `parking_lot::Mutex` to avoid unwrapping and release the lock on poisoning

Reply:
Next time...